main.py

```diff
@@ -818,6 +796,8 @@ def handle_interface_link_down(self, interface):
         Handler for interface link_down events
         """
         for evc in self.get_evcs_by_svc_level():
+            if evc.archived:
```

Comment:
Same comment as the previous thread, now it no longer should need this check here

Reply:
Removed in favor of a lock.